formats/polus-ome-zarr-converter-plugin/src/main.py

```diff
@@ -127,6 +138,17 @@ def main(
         default=".*",
     )
 
+    # Input arguments
+    parser.add_argument(
+        "--fileExtension",
+        dest="fileExtension",
+        type=str,
+        help="Type of data conversion",
+        required=True,
+        default=".*",
```

Comment:
Two things should happen here:
1. This should not be a required argument
2. This should default to `FILE_EXT`

Reply:
I removed required argument. But default value cannot be set to variable FILE_EXT. As we are defining this variable in the function and setting its value that we passed in the fileExtension argument. Should I removed the default value?